packages/botonic-react/src/index.d.ts

```diff
@@ -198,11 +201,37 @@ export class WebchatApp {
   updateWebchatSettings(settings: WebchatSettingsProps): void
 }
 
+export interface WebChatContextProps {
+  sendText: (text: string) => string
+  sendAttachment: (attachment: File) => string
+  sendPayload: (payload: string) => string
+  sendInput: (input: core.Input) => string
+  openWebview: (webviewComponent: Webview) => string
+  addMessage: (message: WebchatMessage) => string
+  updateMessage: (message: WebchatMessage) => string
+  updateReplies: (replies: boolean) => string
+  updateLatestInput: (input: core.Input) => string
+  closeWebview: () => string
+  toggleWebchat: () => string
```

Comment:
it's strange that many of these  returned values are strings when they should not return anything. 
Maybe because in the context are initialized with `''`?

Reply:
right, I'll change it to return void. and will also change the initialState